platforms/shared/tests/test-client/main.cpp

```diff
@@ -125,6 +127,25 @@ void initLogger()
     el::Loggers::reconfigureAllLoggers(conf);
 }
 
+void testUploadCount(prc::Client& client, prc::id_t accountId, prc::id_t instrumentId)
+{
+    prc::Counts counts;
+    time_point ftime = boost::chrono::system_clock::now() - boost::chrono::hours(1);
+
+    for (int i = 0; i < 3; ++i)
+    {
+        ftime += boost::chrono::minutes(1) + boost::chrono::seconds(17);
+        counts.push_back(prc::Count(prc::toTimestamp(ftime), i * 10 + 3));
+    }
+
+    prc::Status status = client.uploadCount(accountId, instrumentId, counts, false);
```

Comment:
How would it be useful? T
The intent of the whole test-client app to provide testing of added functionality, so one adds a short piece of code to see that correct data, when sent to server are accepted. It is not and example, it is not a unit or integration test and the task wasn't about tests or testing. This is why I'm concerned about focus of the code review.